autoload/lsp/utils.vim

```diff
@@ -2,6 +2,13 @@ function! lsp#utils#is_remote_uri(uri) abort
     return a:uri =~# '^\w\+::' || a:uri =~# '^\w\+://'
 endfunction
 
+function! lsp#utils#decode_uri(uri) abort
```

Comment:
let us not make this public yet.

Reply:
Sorry, I'm far from expert in vim script :). Please check that I fixed it right.